tests/roles/mariadb_copy/tasks/main.yaml

```diff
@@ -65,8 +65,8 @@
     oc wait --for condition=Ready pod/mariadb-copy-data --timeout=30s
   register: mariadb_data_pod_result
   until: mariadb_data_pod_result is success
-  retries: 5
-  delay: 6
+  retries: 30
```

Comment:
Just a comment on the change rationale: The original is not 30 seconds as the PR title says. The command has `--timeout=30s` and it is repeated 5 times, with an additional 6 second delay in between. So in total it's like 170-ish seconds

The edit makes it the `--timeout=30s` be repeated 30 times with 2 sec pause, so it's something like ~960 seconds, so 16 minutes. That seems a bit much to me, but it's probably still ok.

Reply:
IT looks like you are right looking at failed task

TASK [mariadb_copy : wait for the pod to come up] ******************************
Saturday 14 September 2024  12:43:24 +0000 (0:00:00.591)       0:13:49.262 **** 
FAILED - RETRYING: [localhost]: wait for the pod to come up (30 retries left).
FAILED - RETRYING: [localhost]: wait for the pod to come up (29 retries left).
FAILED - RETRYING: [localhost]: wait for the pod to come up (28 retries left).
FAILED - RETRYING: [localhost]: wait for the pod to come up (27 retries left).
FAILED - RETRYING: [localhost]: wait for the pod to come up (26 retries left).
FAILED - RETRYING: [localhost]: wait for the pod to come up (25 retries left).
FAILED - RETRYING: [localhost]: wait for the pod to come up (24 retries left).
FAILED - RETRYING: [localhost]: wait for the pod to come up (23 retries left).
FAILED - RETRYING: [localhost]: wait for the pod to come up (22 retries left).
FAILED - RETRYING: [localhost]: wait for the pod to come up (21 retries left).
FAILED - RETRYING: [localhost]: wait for the pod to come up (20 retries left).
FAILED - RETRYING: [localhost]: wait for the pod to come up (19 retries left).
FAILED - RETRYING: [localhost]: wait for the pod to come up (18 retries left).
FAILED - RETRYING: [localhost]: wait for the pod to come up (17 retries left).
FAILED - RETRYING: [localhost]: wait for the pod to come up (16 retries left).
FAILED - RETRYING: [localhost]: wait for the pod to come up (15 retries left).
FAILED - RETRYING: [localhost]: wait for the pod to come up (14 retries left).
FAILED - RETRYING: [localhost]: wait for the pod to come up (13 retries left).
FAILED - RETRYING: [localhost]: wait for the pod to come up (12 retries left).
FAILED - RETRYING: [localhost]: wait for the pod to come up (11 retries left).
FAILED - RETRYING: [localhost]: wait for the pod to come up (10 retries left).
FAILED - RETRYING: [localhost]: wait for the pod to come up (9 retries left).
FAILED - RETRYING: [localhost]: wait for the pod to come up (8 retries left).
FAILED - RETRYING: [localhost]: wait for the pod to come up (7 retries left).
FAILED - RETRYING: [localhost]: wait for the pod to come up (6 retries left).
FAILED - RETRYING: [localhost]: wait for the pod to come up (5 retries left).
FAILED - RETRYING: [localhost]: wait for the pod to come up (4 retries left).
FAILED - RETRYING: [localhost]: wait for the pod to come up (3 retries left).
FAILED - RETRYING: [localhost]: wait for the pod to come up (2 retries left).
FAILED - RETRYING: [localhost]: wait for the pod to come up (1 retries left).
fatal: [localhost]: FAILED! => {"attempts": 30, "changed": true, "cmd": "set -euxo pipefail\n\nexport KUBECONFIG=~/adoption/kubeconfig\n\noc wait --for condition=Ready pod/mariadb-copy-data --timeout=30s\n", "delta": "0:00:30.154755", "end": "2024-09-14 13:00:08.511079", "msg": "non-zero return code", "rc": 1, "start": "2024-09-14 12:59:38.356324", "stderr": "+ export KUBECONFIG=/home/stack/adoption/kubeconfig\n+ KUBECONFIG=/home/stack/adoption/kubeconfig\n+ oc wait --for condition=Ready pod/mariadb-copy-data --timeout=30s\nerror: timed out waiting for the condition on pods/mariadb-copy-data", "stderr_lines": ["+ export KUBECONFIG=/home/stack/adoption/kubeconfig", "+ KUBECONFIG=/home/stack/adoption/kubeconfig", "+ oc wait --for condition=Ready pod/mariadb-copy-data --timeout=30s", "error: timed out waiting for the condition on pods/mariadb-copy-data"], "stdout": "", "stdout_lines": []}

It finished 16 minutes later as you calculated :(

I guess something is wrong with RHEV env as such simple task waits for

Reply:
I made a new change which gives 5 minute in total but better responsiveness from task